docker/php/Dockerfile

```diff
@@ -15,6 +15,8 @@ RUN apt-get update \
 RUN curl 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key' | apt-key add - \
     && echo "deb https://deb.nodesource.com/node_${NODE_VERSION}.x $(lsb_release -cs) main" > /etc/apt/sources.list.d/nodesource.list
 
+RUN curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.deb.sh' | bash
```

Comment:
followed default installation instruction for Ubuntu/Debian from the [official website](https://symfony.com/download)